qa/tasks/cephadm.py

```diff
@@ -1173,14 +1173,18 @@ def add_mirror_to_cluster(ctx, mirror):
     registries_conf = '/etc/containers/registries.conf'
     
     for remote in ctx.cluster.remotes.keys():
-        config = teuthology.get_file(
-            remote=remote,
-            path=registries_conf
-        )
-        new_config = toml.dumps(registries_add_mirror_to_docker_io(config.decode('utf-8'), mirror))
+        try:
+            config = teuthology.get_file(
+                remote=remote,
+                path=registries_conf
+            )
+            new_config = toml.dumps(registries_add_mirror_to_docker_io(config.decode('utf-8'), mirror))
 
-        teuthology.sudo_write_file(
-            remote=remote,
-            path=registries_conf,
-            data=new_config,
-        )
+            teuthology.sudo_write_file(
+                remote=remote,
+                path=registries_conf,
+                data=new_config,
+            )
+        except FileNotFoundError as e:
+            # Docker doesn't ship a registries.conf
+            log.info('Failed to add mirror: %s' % str(e))
```

Comment:
Isn't it an error and we should stop testing now?

Reply:
if the file isn't there, we can simply fall back to the default config and download from docker.io

Reply:
I understand you guys are in a rush that you merged it. But since you were trying to override the non-existing file, it is definitely an error, at least, you might want to use `log.warning`... ;-) otherwise I don't see a reason to inform user that is missing anyways.
PS: The PR description does not say much to me.